components/tooltip/tooltip.js

```diff
@@ -749,7 +777,11 @@ class Tooltip extends RtlMixin(LitElement) {
 		}
 	}
 
-	_onTargetFocus() {
+	async _onTargetFocus() {
+		if (this.onlyShowIfTruncating !== undefined) {
```

Comment:
I _think_ this could just be done in the resize observer rather than on each focus/hover, since that seems to run even if the tooltip isn't visible

Reply:
Hmm, I wonder if a combination of both would work well.  We didn't really want to do the measuring/calculating on resize for all these tooltips in case they never actually open the tooltips, but if the page hasn't changed size there's no reason to recalculate again.  I wonder if we measure on hover/focus, but then don't remeasure if the resizeObserver hasn't fired since?

Reply:
ah that's a good point and good idea - I'll test that out